Sources/Networking/HTTPClient/HTTPResponse.swift

```diff
@@ -86,7 +86,7 @@ extension ErrorResponse {
 
         return ErrorUtils.backendError(
             withBackendCode: self.code,
-            backendMessage: self.message,
+            message: self.attributeErrors.description,
```

Comment:
Hmm are we still using the `backendMessage` parameter anywhere? I couldn't find a usage of it but I might be wrong.

Reply:
Oh good catch! I changed this before I fixed #1776 and forgot to add it back. Thanks for the thorough review 👏🏻